.gitignore

```diff
@@ -19,3 +19,6 @@ result-*
 
 # generated by pkgs/common-updater/update-script.nix
 update-git-commits.txt
+
+# jetbrains IDEs
+.idea/
```

Comment:
Rather than put this in the shared `.gitingore` file, it would be a better to ignore it locally by putting it in `.git/info/exclude'.

Reply:
I'm using the same repo on multiple laptops/machines, and manual handling of these sounds not very optimal, unless there's a very good reason not to share this ignore rule universally. In what situation is it not advisable to ignore `.idea` folder?

Reply:
I don't mind having it, but in general I find it more convenient to add the ignores for IDE's that I regularly use in my global gitignore file - see for example https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
I find these two gitignores serve two distinct purposes - the one on the user side says "these are the files that I don't want to see checked in any of the repositories I may work on", wheres the one in the repository says "these are the files that this repo universally does not want to see from anyone", and I think this repo definitely doesn't want to see jetbrains-workspace-specific files.

Anyhow, I think I can ignore it myself, it's not that critical.